### PR TITLE
Update WebPage.js

### PR DESCRIPTION
**Pull request by JoelJaison394**
Solved the problem off , senting  Random messages when a Message arrived in the inbox . I solved this by giving more specified selection of accept button So no other buttons gets selected .

### CODE REVIEW

**Comment by kumarRaj-Gupta** [comment]
@JoelJaison394 Nice one bro . I arrived lately , mb . Are there any other issues in this repo you aware  of ?

**Comment by akshaykhanapuri** [comment]
It looks like changing the classname to `button.artdeco-button--secondary.invitation-card__action-btn` is breaking the bot with the following error: 

```
TimeoutError: Looking for element
Wait timed out after 15133ms
    at /Path_to_project/linkedin-bot/node_modules/selenium-webdriver/lib/webdriver.js:929:17
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  remoteStacktrace: ''
```
But the idea looks solid. I was able to work around this by replacing the `.` with a `space`
Ref: https://github.com/akshaykhanapuri/linkedin-bot/blob/master/WebPage.js#L67

**Comment by hkirat** [comment]
@JoelJaison394 were you able to confirm if this works? ref https://github.com/hkirat/linkedin-bot/pull/1#issuecomment-1454832360

File: WebPage.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const BasePage = function (customAudio = null) {
   }
 
   this.pressAcceptButton = async function () {
-    let buttons = await this.findByClassName("artdeco-button--secondary");
+    let buttons = await this.findByClassName("button.artdeco-button--secondary.invitation-card__action-btn");
```
Comment (laukik-target) [inline WebPage.js:63]: I feel a better way would be using XPath rather by class name

**Comment by amanzrx4** [comment]
just use xpath //*[@id="ember1830"]

**Pull request closed without merging**